lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -49,15 +54,17 @@ def validate_upload_document(lighthouse_document)
   # return [Faraday::Response] BenefitsDocuments::WorkerService makes http
   # calls with the Faraday gem under the hood
   def submit_upload_document(lighthouse_document, file_body)
-    BenefitsDocuments::Form526::UploadSupplementalDocumentService.call(file_body, lighthouse_document)
+    api_response = BenefitsDocuments::Form526::UploadSupplementalDocumentService.call(file_body, lighthouse_document)
+    handle_lighthouse_response(api_response)
   end
 
-  def log_upload_success(uploading_class_prefix)
-    StatsD.increment("#{uploading_class_prefix}.#{STATSD_PROVIDER_METRIC}.#{STATSD_SUCCESS_METRIC}")
-  end
+  # def log_upload_success(uploading_class_prefix)
+  #   StatsD.increment("#{uploading_class_prefix}.#{STATSD_PROVIDER_METRIC}.#{STATSD_SUCCESS_METRIC}")
+  # end
 
-  def log_upload_failure(uploading_class_prefix, error_class, error_message)
-    StatsD.increment("#{uploading_class_prefix}.#{STATSD_PROVIDER_METRIC}.#{STATSD_FAILED_METRIC}")
+  # def log_upload_failure(uploading_class_prefix, error_class, error_message)
+  def log_upload_failure(error_class, error_message)
+    StatsD.increment("#{@statsd_metric_prefix}.#{STATSD_PROVIDER_METRIC}.#{STATSD_FAILED_METRIC}")
 
     Rails.logger.error(
       'LighthouseSupplementalDocumentUploadProvider upload failure',
```

Comment:
We need more info in this error, such as at the very least, `submission_id` and `user_uuid`, every log message should be able to tie back to a submission_id.